packages/cli/src/commands/install.ts

```diff
@@ -1,19 +1,17 @@
-import { installNode } from "../installers/node"
-
 /**
  * Usage: @appsignal/cli install [options]
  *
  * installs AppSignal
  *
  * Options:
  * -h, --help  display help for command
+ * @deprecated
  */
 export async function install() {
-  console.log("🚀 Alright! Let's install AppSignal to your project!")
-
   try {
-    await installNode(process.cwd())
-    console.log("✅ Done!")
+    console.log(
+      "CLI installer is deprecated. Please use the web wizard instead."
```

Comment:
There should probably be a link here to the AppSignal app creation screen. I don't think it will be obvious to users running this command what "the web wizard" is.